LPS-63706 Change group and name of portal artifacts

Ithildir opened the pull request:
Hi!
Can you please publish a new `gradle-plugins`?

Thank you!

Ithildir committed:
LPS-63706 Update to support the new group and name pattern

Ithildir committed:
LPS-63706 Update Gradle

Ithildir committed:
LPS-63706 Change group and name while publishing the portal artifacts

Ithildir committed:
LPS-63706 SF

Ithildir committed:
LPS-63706 Task cache

Ithildir committed:
LPS-63706 Update portal lib names included in Bnd files

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/1002/">test-portal-acceptance-pullrequest(master)</a></p>

Ithildir committed:
LPS-63706 Ignore cached portal artifacts from Git

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 54 minutes 8 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/45b742134270f1bb2a4a997263b356658fab5bf6">45b742134270f1bb2a4a997263b356658fab5bf6</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/1002/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19067/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20492/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16902/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19916/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19200/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16903/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27618/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23230/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19068/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28134/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20815/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20816/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20817/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20818/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27604/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19069/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20819/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20820/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20849/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20494/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31390/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26794/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20057/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20058/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23229/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20848/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20821/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19070/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19069/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk7</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20822/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk7</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20850/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19070/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31097/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19464/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23354/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk7</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26795/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20497/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23231/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk7</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23232/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk7</a></li><li><strike><strong><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31391/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></strong></strike></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/2240/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-11.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1002/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest(master)/1002/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>40 Jobs Passed.<br />2 Jobs Failed.</p><pre>Completed with the status of FAILURE.</pre></li><li><h5 job-result="FAILURE"><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31391/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></h5><h6>Job Results:</h6><p>1 Test Passed.<br />1 Test Failed.</p><ol><li><strong><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=0,label_exp=!master/31391/">test-portal-acceptance-pullrequest-batch&#40;master&#41; ?? 0,!master #31391</a></strong><pre><br />     &#91;exec&#93;       http://repository.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.portal-test/7.0.0-SNAPSHOT/com.liferay.portal-test-7.0.0-SNAPSHOT.pom<br />     &#91;exec&#93;       http://repository.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.portal-test/7.0.0-SNAPSHOT/com.liferay.portal-test-7.0.0-SNAPSHOT.jar<br />     &#91;exec&#93;   Required by:<br />     &#91;exec&#93;       com.liferay:portal-app-license-test:1.0.0<br />     &#91;exec&#93; &#62; Could not find com.liferay:com.liferay.portal-test-internal:7.0.0-SNAPSHOT.<br />     &#91;exec&#93;   Searched in the following locations:<br />     &#91;exec&#93;       file:/root/.m2/repository/com/liferay/com.liferay.portal-test-internal/7.0.0-SNAPSHOT/maven-metadata.xml<br />     &#91;exec&#93;       file:/root/.m2/repository/com/liferay/com.liferay.portal-test-internal/7.0.0-SNAPSHOT/com.liferay.portal-test-internal-7.0.0-SNAPSHOT.pom<br />     &#91;exec&#93;       file:/root/.m2/repository/com/liferay/com.liferay.portal-test-internal/7.0.0-SNAPSHOT/com.liferay.portal-test-internal-7.0.0-SNAPSHOT.jar<br />     &#91;exec&#93;       http://repository.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.portal-test-internal/7.0.0-SNAPSHOT/maven-metadata.xml<br />     &#91;exec&#93;       http://repository.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.portal-test-internal/7.0.0-SNAPSHOT/com.liferay.portal-test-internal-7.0.0-SNAPSHOT.pom<br />     &#91;exec&#93;       http://repository.liferay.com/nexus/content/groups/public/com/liferay/com.liferay.portal-test-internal/7.0.0-SNAPSHOT/com.liferay.portal-test-internal-7.0.0-SNAPSHOT.jar<br />     &#91;exec&#93;   Required by:<br />     &#91;exec&#93;       com.liferay:portal-app-license-test:1.0.0<br />     &#91;exec&#93; <br />     &#91;exec&#93; * Try:<br />     &#91;exec&#93; Run with --info or --debug option to get more log output.<br />     &#91;exec&#93; <br />     &#91;exec&#93; * Exception is:<br />     &#91;exec&#93; org.gradle.api.artifacts.ResolveException: Could not resolve all dependencies for configuration &#39;:core:portal-app-license-test:portalTest&#39;.<br />     &#91;exec&#93; &#09;at org.gradle.api.internal.artifacts.ivyservice.DefaultLenientConfiguration.rethrowFailure&#40;DefaultLenientConfiguration.java:62&#41;<br />     &#91;exec&#93; &#09;at org.gradle.api.internal.artifacts.ivyservice.DefaultResolvedConfiguration.rethrowFailure&#40;DefaultResolvedConfiguration.java:36&#41;<br />     &#91;exec&#93; &#09;at org.gradle.api.internal.artifacts.ivyservice.SelfResolvingDependencyConfigurationResolver$FilesAggregatingResolvedConfiguration.rethrowFailure&#40;SelfResolvingDependencyConfigurationResolver.java:112&#41;</pre></li></ol></li></ol>

Ithildir committed:
LPS-63706 Publish portal-test* artifacts before building "core" modules

The "core" dir contains a few "*-test" modules, which need those artifacts in order to be built: the dependency is added by default in LiferayDefaultsPlugin and, if a module is a "*-test" one, building the jar requires building the test classes. We can bypass this problem in two ways: publishing the portal-test* artifacts, or excluding the "core/*-test" modules.

Ithildir committed:
LPS-63706 FAKE GRADLE CACHE

Ithildir committed:
LPS-63706 Use latest gradle-plugins

Ithildir commented:
ci:retest

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/1033/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/1014/">test-portal-acceptance-pullrequest(master)</a></p>

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 47 minutes 15 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/aca0a9ab2ac24c6588aaebd61220674e6848a51e">aca0a9ab2ac24c6588aaebd61220674e6848a51e</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/1033/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20864/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19096/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16935/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23380/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27647/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27624/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27648/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16937/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20518/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31132/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19225/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19936/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16938/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20519/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19935/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31531/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19937/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31133/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20095/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19100/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31530/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31436/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27936/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27937/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27646/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27938/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27650/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27649/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20869/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16939/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk7</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20096/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23384/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk7</a></li><li><strike><strong><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19108/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></strong></strike></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19489/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31438/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19490/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26826/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk7</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19940/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk7</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31135/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk7</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19943/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/1147/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-18.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1033/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest(master)/1033/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>40 Jobs Passed.<br />2 Jobs Failed.</p><pre>Completed with the status of FAILURE.</pre></li><li><h5 job-result="UNSTABLE"><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19108/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></h5><h6>Job Results:</h6><p>4292 Tests Passed.<br />3 Tests Failed.</p><ol><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/19108//testReport/com.liferay.shopping.service.persistence.test/ShoppingOrderPersistenceTest/testActionableDynamicQuery">ShoppingOrderPersistenceTest.testActionableDynamicQuery</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/19108//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">PortalLogAssertorTest.testScanXMLLog</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch%28master%29/AXIS_VARIABLE=4,label_exp=!master/19108//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanOSGiLog">PortalLogAssertorTest.testScanOSGiLog</a></li></ol></li></ol>

liferay-continuous-integration commented:
<h1>All tests PASSED!</h1><p>Build Time: 53 minutes 36 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/aca0a9ab2ac24c6588aaebd61220674e6848a51e">aca0a9ab2ac24c6588aaebd61220674e6848a51e</a></p><h4>Job Summary:</h4><ul><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/1014/">test-portal-acceptance-pullrequest(master)</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27939/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16936/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-hypersonic20-jdk7/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23381/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/0</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31532/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31437/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/10</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28143/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/11</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31533/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/12</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31439/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/13</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19491/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/14</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19939/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/15</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19941/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/16</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19942/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/17</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20521/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/18</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20522/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/19</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19938/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/2</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20523/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/20</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27655/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/21</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16941/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/22</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27656/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/23</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20520/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/3</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27652/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/4</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27626/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/5</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19106/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/6</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/23383/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16940/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19107/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat8-mysql56-jdk7/9</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20879/">test-portal-acceptance-pullrequest-batch(master)/functional-wildfly10-mariadb100-jdk8/0</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/16942/">test-portal-acceptance-pullrequest-batch(master)/integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27941/">test-portal-acceptance-pullrequest-batch(master)/integration-mysql56-jdk7</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/28144/">test-portal-acceptance-pullrequest-batch(master)/integration-postgresql94-jdk7</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31534/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-hypersonic20-jdk7</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/26827/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql56-jdk7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20524/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-postgresql94-jdk7</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19109/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk7</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/27627/">test-portal-acceptance-pullrequest-batch(master)/plugins-compile-jdk7</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/31441/">test-portal-acceptance-pullrequest-batch(master)/portal-web-jdk7</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19227/">test-portal-acceptance-pullrequest-batch(master)/ruby-sass-compiler-jdk7</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19945/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/19102/">test-portal-acceptance-pullrequest-batch(master)/tck-jdk7</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/20097/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk7</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/2329/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-20.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/1014/jenkins-report.html">here</a>.</h5>

brianchandotcom commented:
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/1959b99818...73e432c21a
